src/GetDocumentInsider/AnsiTextWriter.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Extensions.ApiDescription.Tool
+{
+    internal class AnsiTextWriter
```

Comment:
Minor: Shouldn't this be disposable?

Reply:
Yes though you're also right it's a minor issue because this is a short-lived tool. In addition, `AnsiTextWriter` should be removed as part of #8566.

I filed the low-priority issue #8592 in case we decide to make that small fix in advance of doing #8566.

Reply:
In general no, this should not be disposable. If you didn't create it, you probably shouldn't dispose it. The inner `TextWriter` is passed in. This is an internal utility and it's always used with the Console streams, which shouldn't be disposed.